lib/cli/writer.js

```diff
@@ -67,6 +69,15 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && !constraints.isDriveLargeEnough(drive, {
+        size: image.size.final.value
+      })) {
+        throw errors.createUserError(
+          'The image you selected is too big for this drive',
+          'Plug a bigger drive and try again'
```

Comment:
I also thought the "Plug..." wording was strange, but I didn't bother commenting because Juan mentioned he'd be refactoring all the error messages soon anyway ;-)